Docs: Initial Sphinx add (#3)
cobbler/libcobblersignatures: docs/quickstart.rst

**********
Quickstart
**********

Installing from PyPi: ``pip install libcobblersignatures``

Running the CLI when installed: ``cobbler-manage-signatures``

What is this library for?
#########################

> The purpose of this library is to manage and abstract the access to how we manage the data for our operating system
> distribution detection. Currently this is a file which is saved to the disk in the json format.

Reviewer: ```suggestion
The purpose of this library is to manage and abstract the access on how we manage the data for our operating system
detection. Currently this is a JSON file which is saved to the hard drive.
```

> To understand what this library does we need to go into detail about what Cobbler does with the provided data. Cobbler
> has a functionality called ``import``, which means that you give the CLI or API an ISO file which then gets searched
> through to save information in Cobbler about this ISO. This means that Cobbler somehow needs to detect what operating
> system and which version of the system is there. The data to differentiate between the ISOs is coming from this library.

Reviewer: ```suggestion
To understand what this library does, we need to go into detail about what Cobbler does with the provided data. Cobbler
has a functionality called ``import``, which means that you give the CLI or API an ISO image which then gets searched
through to save information about it in Cobbler. This means that Cobbler somehow needs to detect which operating
system and which version of it is inside the ISO image. This library provides the data to distinguish between these ISOs from each other.
```

How to use it?
##############

> Up top you see the most basic usage of this library. To get to most of it, I recommend not installing this library
> directly but rather with Cobbler who has a dependency on this. If you just want to manipulate the ``signatures.json``
> file I would recommend using - if available - a built package from your package manager.

Reviewer: ```suggestion
In the CLI section you can see the most basic usage of this library. To get the most out of it, it's not recommend to install this library
directly but rather use it together with Cobbler who has a dependency on it. If you just want to manipulate the ``signatures.json``
file, it's recommended to use a built package from your package manager (if available).
```
